static/app/views/organizationStats/index.tsx

```diff
@@ -403,20 +407,8 @@ export class OrganizationStats extends Component<Props> {
 
 export default withPageFilters(withOrganization(OrganizationStats));
 
-const PageGrid = styled('div')`
-  display: grid;
-  grid-template-columns: 1fr;
-  gap: ${space(2)};
-
-  @media (min-width: ${p => p.theme.breakpoints.small}) {
-    grid-template-columns: repeat(2, 1fr);
-  }
-  @media (min-width: ${p => p.theme.breakpoints.large}) {
-    grid-template-columns: repeat(4, 1fr);
-  }
-`;
-
 const DropdownDataCategory = styled(CompactSelect)`
+  position: relative;
```

Comment:
This CSS is just to add the vertical line to left side of the category selector.